src/include/net/l3/icmpv6.h

```diff
@@ -134,10 +134,10 @@ typedef struct icmp6hdr {
 		struct ndpbody_router_advert router_advert;
 		struct ndpbody_neighbor_solicit neighbor_solicit;
 		struct ndpbody_neighbor_advert neighbor_advert;
-	} __attribute__((packed)) body[];
+	} __attribute__((packed)) body;
 } __attribute__((packed)) icmp6hdr_t;
 
-#define ICMP6_MIN_HEADER_SIZE (sizeof(struct icmp6hdr))
+#define ICMP6_MIN_HEADER_SIZE (4)
```

Comment:
I think it's a good idea to write comment why it's 4 (`sizeof(struct icmp6hdr)` without body).

Reply:
thanks. done